tests/fixtures/contract-fixtures.ts

```diff
@@ -0,0 +1,9 @@
+import { contractSources } from "../tests/constants/contractSources";
+import { getCompiled } from "../tests/util/contracts";
+
+exports.mochaGlobalSetup = async function () {
+  // First compile all contracts
+  console.log("Making sure all contracts are compiled...");
```

Comment:
Do we want to keep those logs ?

Reply:
I like them. But I can remove them if you want